pkg/restapi/vc/operation/operations.go

```diff
@@ -492,7 +492,7 @@ func (o *Operation) storeVCHandler(rw http.ResponseWriter, req *http.Request) {
 func (o *Operation) storeVC(data *StoreVCRequest, vc *verifiable.Credential, rw http.ResponseWriter) {
 	doc, err := o.buildStructuredDoc(data, vc)
 	if err != nil {
-		o.writeErrorResponse(rw, http.StatusBadRequest, err.Error())
+		o.writeErrorResponse(rw, http.StatusInternalServerError, err.Error())
```

Comment:
Are we validating VC in this function? If so maybe it makes more sense to leave it as BadRequest. For cases bellow I agree they should be InternalServerError

Reply:
I agree, done